src/lib/tests/locale.test.ts

```diff
@@ -1,3 +1,6 @@
+/**
+ * @jest-environment node
+ */
```

Comment:
Using the `node` testing environment should provide a more realistic environment for main process tests. Also, the tests should run faster.